Rules/CommonScripts/HelpfulDeathTips.cfg

```diff
@@ -34,7 +34,7 @@ tips = To hide these tips, and other in-game help, hit F1.;
 		A bomb can deal up to 3 hearts of damage.;
 		Drills only yield 75% of stone, but 100% of gold.;
 		Good players can play any class when needed.;
-		Make sure you're helping your team advance; don't just hide out in the back!;
+		Make sure you're helping your team advance. Don't just hide out in the back!;
```

Comment:
The edit on line 8 was made to make the tip more generic so it could be shown in the HUD and shown in chat.

The edit on line 37 is necessary because the semicolon caused the tip to be split into two separate tips.

I have updated all the locales to allow them to work with the updated translations.